test-suite/tests/testw90_bvec/lead.win

```diff
@@ -0,0 +1,98 @@
+! Lead : Tutorial Example 2
+
+ num_wann        =   4 
+ num_iter        = 20 
+
+ write_hr        = .true.
```

Comment:
I would make a test where only `write_bvec` is set, and all other plotting options are removed (`write_hr`, `write_rmn`) - see also my other comment above.